README.md

```diff
@@ -272,6 +277,15 @@ curl -L https://github.com/yt-dlp/yt-dlp/releases/latest/download/yt-dlp -o ~/.l
 chmod +x ~/.local/bin/yt-dlp
 ```
 
+##### Install [patch](https://savannah.gnu.org/projects/patch/)
+
+```
+mkdir ~/.patch
+curl -o ~/.patch/patch.tar.zst https://mirror.sunred.org/archlinux/core/os/x86_64/patch-2.7.6-10-x86_64.pkg.tar.zst
+tar xvf .patch/patch.tar.zst -C ~/.patch/
+cp ~/.patch/usr/bin/patch ~/.local/bin/
+```
+
```

Comment:
Can you please add a note here that this is for `-U` / for updating?

Reply:
Ive added your proposal to the "dependencies section"

Reply:
Oh i see what you mean.
Ill add it